{{cookiecutter.projectDirectory}}/{{cookiecutter.projectName}}Kit/Code/Utilities/Decoders.swift

```diff
@@ -11,11 +11,11 @@ public struct Decoders {
         let container = try decoder.singleValueContainer()
         let raw = try container.decode(String.self)
 
-        if let value = Formatters.isoDateFormatter.date(from: raw) {
+        if let value = Formatters.Date.isoDate.date(from: raw) {
```

Comment:
Ich denke, dass wir keine formatters in projekt brauchen, da wir auch gleiche im toolbox haben... check nochmals aber sie sind gerade überflüssig

Reply:
Ich würde die aus der Toolbox raus nehmen und in das Starter-Projekt mit rein nehmen. 
Formatter sind sehr Projekt abhängig und sollten leicht erweiterbar sein.

Reply:
hier handelt sich um standard formatters weshalb ich denke dass sie in toolbox richtig sind und wenn man ein extra braucht sollte selber extension bzw. formatter erstellen... ist nur meine meinung 😅